firebase-ai/src/main/kotlin/com/google/firebase/ai/type/ImagenEditMode.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.firebase.ai.type
+
+/** Represents the edit mode for Imagen */
+public class ImagenEditMode private constructor(internal val value: String) {
+
+  public companion object {
```

Comment:
Redundant phrase:

"Inpainting insertion is used when inserting a new element into an image"

"Inserts a new element into an image"

Same for the other fields in this object.